INSTALL.md

```diff
@@ -67,17 +67,17 @@ git clone https://github.com/onnx/onnx.git
 cd onnx
 git submodule update --init --recursive
 # prefer lite proto
-set CMAKE_ARGS=-DONNX_USE_LITE_PROTO=ON
+set CMAKE_ARGS='-DONNX_USE_LITE_PROTO=ON -DONNX_USE_PROTOBUF_SHARED_LIBS=ON'
```

Comment:
Why is this needed?

Reply:
The previous phrase recommends users to use DONNX_USE_PROTOBUF_SHARED_LIBS=on and exports CMAKE_ARGS='DONNX_USE_PROTOBUF_SHARED_LIBS=on'. However, it is overwritten here. So it's necessary to append that option.

Reply:
There is no way to set multiple options separately with CMAKE_ARGS and use them together.